File: module-1/12_Polymorphism/lecture-student/dotnet/Lecture/Farming/Chicken.cs
using System;

namespace Lecture.Farming
{
    public class Chicken : FarmAnimal
    {
        public Chicken() : base("Chicken", "cluck")
        {
        }

        public void LayEgg()
        {
            Console.WriteLine("Chicken laid an egg!");
        }
    }
}

File: module-1/13_Inheritance_Abstract_Classes/lecture-final/java/src/main/java/com/techelevator/farm/Chicken.java
package com.techelevator.farm;

import java.math.BigDecimal;

public class Chicken extends Livestock {

	public Chicken() {
		super("Chicken");
	}

	@Override
	public String makeSound() {
		return "Cluck";
	}

	@Override
	public BigDecimal getPrice() {
		return new BigDecimal("20.00");
	}

	public void layEgg() {
		System.out.println("Chicken laid an egg!");
	}

}